bundles/org.openhab.binding.knx/README.md

```diff
@@ -72,8 +73,7 @@ This is however an experimental feature, very prone to the actual on the KNX bus
 | readInterval | N        | Interval (in seconds) to actively request reading of values from the bus (0 if they should only be read once at startup) | 0                                                                           |
 
 Different kinds of channels are defined and can be used to group together Group Addresses.
-All channel types share two configuration parameters: *read*, an optional parameter to indicate if the 'readable' group addresses of that Channel should be read at startup (default: false), and *interval*, an optional parameter that defines an interval between attempts to read the status group address on the bus, in seconds.
-When defined and set to 0, the interval is ignored (default: 0)
+All channels of a device share one configuration parameter defined on device level: *readInterval*, an optional parameter which indicates if 'readable' group addresses of that Channel should be read periodically at the given interval, in seconds. 'Readable' group addresses are marked with an `<` in the group address definition of a Channel, see below. All readable group addresses and are queried by openHAB during startup. If readInterval is not specified or set to 0, no further periodic reading will be triggered (default: 0).
```

Comment:
Looks like "and" is not expected in "All readable group addresses and are queried by "